docs/threat-modeling-for-designers/threat-modeling-for-designers.md

```diff
@@ -0,0 +1,57 @@
+# Threat Modeling for Designers
+
+This is a living, work-in-progress documentation to shape how might _Threat Modeling_ (borrowing from security field) look like for designers. Alternatively, we can relabel "threat modeling" as "risk assessment."
+
+We welcome collobration on this documentation — Suggest and create a PR to add, suggest improvement, etc.
+
+## Why
+
+As a framework, _threat modeling for designers_ will help designers:
+
+1. Understand the threats users face and how designers can counter those threats
+2. Make security more manageable
+3. Collobrate with partner(s) and colleagues to develop an understanding of our/their assets, adversaries, and capaiblities
```

Comment:
`Adversaries` stood out to me here and feels pretty opinionated that these actors are `bad`. Is that something the typical project would face? Or is this for a particular type of project (like security)?

Reply:
> Adversaries stood out to me here and feels pretty opinionated that these actors are bad

Can you say more? Initially, my thinking is that it would be helpful to be opinionated here with "adversaries" to address specific scenarios instead of being too broad / open ended that we veer off too much into speculation; however, I can see how the former's narrow approach, esp. coupled with our biases, could do us a disservice by overlooking the others. 

> Is that something the typical project would face? Or is this for a particular type of project (like security)

This is something I'm still finding out and would like to hear your take on this. I imagine that a typical project should factor in threats and harms like how we factor in accessibility and inclusive design in our project.